stixcore/products/product.py

```diff
@@ -743,6 +768,28 @@ def fits_daily_file(self):
         return False
 
 
+class CountDataMixin:
+    @property
+    def dmin(self):
+        return self.data['counts'].min().value
+
+    @property
+    def dmax(self):
+        return self.data['counts'].max().value
+
+    @property
+    def bunit(self):
+        return 'counts'
+
+    @property
+    def exposure(self):
+        return self.data['timedel'].as_float().min().to_value('s')
```

Comment:
might resolve the test errors not sure
```suggestion
        return self.data['timedel'].min().to_value('s').as_float()
```

Reply:
fixed